lib/project_types/script/commands/javy.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+require "shopify_cli"
+require_relative "../../../../ext/javy/javy.rb"
+
+module Script
+  class Command
+    class Javy < ShopifyCLI::SubCommand
+      hidden_feature
```

Comment:
This is intended to hide the command from automatically appearing in the help text, but there is a bug in the core help code. I'll open another PR to fix it!